container-core/src/main/java/com/yahoo/container/core/documentapi/DocumentAccessProvider.java

```diff
@@ -0,0 +1,38 @@
+package com.yahoo.container.core.documentapi;
```

Comment:
Add copyright header to your intellij template :)

Reply:
Done!